supportingFunctions.R

```diff
@@ -0,0 +1,130 @@
+####R Project
+####Supporting Functions
+####Shannon Culbert and Camden Arnold
+####All code assumes working directory is set to RProject directory
+
+################################################################################
+
+###Part 1: Converting all .txt files into .csv files
+#Convert all .txt files to .csv files using for loop
+
+convert_to_csv <- function(dir) {
+  txt_files <- list.files(dir,pattern=".txt") #lists .txt files that need converted
+  for (i in txt_files) {
+    file_path <- file.path(dir,i)
+    newfile_name <- sub(".txt",".csv",i)
+    file_path2 <- file.path(dir,newfile_name)
+    data <- read.table(file=file_path, header=TRUE, sep="") #reads .txt file
+    write.csv(data,file=file_path2,row.names=FALSE,col.names=FALSE) #converts to .csv
+  }
+}
+#Country X has .csv files already
+#Country Y needs .txt files to be converted to .csv files
+#run function to convert Country Y's .txt files to .csv files
+#usage: convert_to_csv("countryY")
+
+################################################################################
+
+###Part 2: Converting all .csv files into a single .csv file
+
+#create path to add all of the information into "allData.csv"
+#includes adding columns for Country and DayOfYear
+combine_data <- function(dir1,dir2) { #creates allData.csv to which data will be added
+  path_init <- file.path("countryX","screen_120.csv")
+  data_init <- read.csv(path_init,header=TRUE)
+  data_init$country = "X"
+  data_init$dayofYear = "120"
+  write.csv(head(data_init,n=0), file = "allData.csv",col.names=FALSE, row.names=FALSE)
+  
```

Comment:
You can create a data with empty row by:
data.frame(gender = character(), marker01 = numeric(), ..., marker10 = numeric(), country = character(), dayofYear = numeric())